fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts

```diff
@@ -944,7 +944,7 @@ export class PseudoFWC {
     }, true);
 
     this.stallWarning.sub((v) => {
-      this.fwcOut126.get().setBitValue(17, v);
+      this.fwcOut126.setBitValue(17, v);
```

Comment:
> [EWD] › error TS2551: Property 'setBitValue' does not exist on type 'Arinc429RegisterSubject'. Did you mean 'setValue'? ()
          [EWD] › →  at PseudoFWC.ts:951:21

Reply:
Took the freedom to fix 🧷 , probably got lost on restoring the branch

Reply:
thanks!